docs/src/index.md

```diff
@@ -1,6 +1,81 @@
 ![GigaSOM.jl](https://prince.lcsb.uni.lu/GigaSOM.jl/img/logo-GigaSOM.jl.png?maxAge=0)
```

Comment:
I think part of this could also go to the README file.

Reply:
And then part of it could be read in here (no duplication of text)

Reply:
ok, we can do that separation once the text is complete so we can decide exactly what goes where

Reply:
yup!